src/providers/DevicesProvider/VideoInputProvider.tsx

```diff
@@ -68,14 +68,15 @@ const VideoInputProvider: React.FC = ({ children }) => {
         return;
       }
 
-      const devices = await audioVideo.listVideoInputDevices();
+      const devices = await audioVideo.listVideoInputDevices(true);
 
       if (isMounted) {
         setVideoInputs(devices);
         audioVideo.addDeviceChangeObserver(observer);
       }
     }
 
+    meetingManager.invokeVideoInputProvider = initVideoInput;
```

Comment:
This seems like it will leak badly.

Reply:
Updated the invoke approach to be based on observer pattern. Removed those `invoke*` callbacks.